site/_docs/installation.md

```diff
@@ -31,16 +32,18 @@ earlier, for CoffeeScript support).
   </p>
 </div>
 
-## Install with RubyGems
+## Install with Bundler and RubyGems
 
-The best way to install Jekyll is via
+The best way to install Jekyll is using a Gemfile via Bundler
 [RubyGems](http://rubygems.org/pages/download). At the terminal prompt,
 simply run the following command to install Jekyll:
 
 {% highlight shell %}
-$ gem install jekyll
+$ gem install bundler
```

Comment:
Only bundler need be installed outside the Gemfile.  Jekyll can stay within it, but we'll have to update and backtrack to make sure everything uses `bundle exec` that way we've consistent documentation.